From d837c1b223db977719e0111e6903333281e924cf Mon Sep 17 00:00:00 2001
From: Mark Salyzyn <salyzyn@google.com>
Date: Mon, 2 Dec 2019 17:02:48 -0800
Subject: [PATCH 592/830] ANDROID: gki_defconfig: IIO=y
X-NVConfidentiality: public

Set the default for CONFIG_IIO=y, a requirement for bandwidth,
latency and proper synchronization of the sensor hub with camera.

Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Bug: 145377190
Change-Id: If4d6b09cc4fc4807404311d793713557ee88f66b
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 12359c34dfa1..868951152439 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -378,6 +378,7 @@ CONFIG_DEVFREQ_GOV_POWERSAVE=y
 CONFIG_DEVFREQ_GOV_USERSPACE=y
 CONFIG_DEVFREQ_GOV_PASSIVE=y
 CONFIG_EXTCON=y
+CONFIG_IIO=y
 CONFIG_PWM=y
 CONFIG_QCOM_PDC=y
 CONFIG_GENERIC_PHY=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 861dac98bb64..53edd7b99449 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -318,6 +318,7 @@ CONFIG_ION=y
 CONFIG_ION_SYSTEM_HEAP=y
 CONFIG_ION_SYSTEM_CONTIG_HEAP=y
 CONFIG_PM_DEVFREQ=y
+CONFIG_IIO=y
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_EXT4_FS=y
-- 
2.17.1

